oracle/src/feeds/blockcypher.rs

```diff
@@ -0,0 +1,64 @@
+use super::{get_http, BitcoinFeed};
+use crate::Error;
+use async_trait::async_trait;
+use clap::Parser;
+use reqwest::Url;
+
+#[derive(Parser, Debug, Clone)]
+pub struct BlockCypherCli {
+    /// Fetch the bitcoin fee estimate from BlockCypher (https://api.blockcypher.com/v1/btc/main).
+    #[clap(long)]
+    blockcypher_url: Option<Url>,
+}
+
+pub struct BlockCypherApi {
+    url: Url,
+}
+
+impl Default for BlockCypherApi {
+    fn default() -> Self {
+        Self {
+            // Mainnet: https://api.blockcypher.com/v1/btc/main
+            // Testnet: https://api.blockcypher.com/v1/btc/test3
```

Comment:
Should we support this on our testnets?

Reply:
This comment is more for reference, this will work on our testnets if the URL is set via the CLI.